cmake/onnxruntime_providers.cmake

```diff
@@ -549,11 +549,16 @@ if (onnxruntime_USE_OPENVINO)
 
     # Library paths
     list(APPEND OPENVINO_LIB_DIR_LIST $ENV{INTEL_OPENVINO_DIR}/deployment_tools/inference_engine/external/tbb/lib)
+    list(APPEND OPENVINO_LIB_DIR_LIST $ENV{INTEL_OPENVINO_DIR}/deployment_tools/inference_engine/external/tbb/bin)
     list(APPEND OPENVINO_LIB_DIR_LIST $ENV{INTEL_OPENVINO_DIR}/deployment_tools/inference_engine/external/mkltiny_lnx/lib)
 
     # Lib names
     if (WIN32)
-      list(APPEND OPENVINO_LIB_LIST inference_engine.lib inference_engine_legacy.lib tbb.lib ${PYTHON_LIBRARIES})
+      if (CMAKE_BUILD_TYPE STREQUAL "Debug")
+        list(APPEND OPENVINO_LIB_LIST inference_engined.lib inference_engine_legacyd.lib tbb.lib ${PYTHON_LIBRARIES})
```

Comment:
Does tbb have to versions too?

Reply:
there was not a tbbd.lib in the official release package of openvino, since it did not threw an error, we have left it to be the same